pkg/sliceutils/concat.go

```diff
@@ -0,0 +1,15 @@
+package sliceutils
+
+// Concat concatenates slices, returning a slice with newly allocated backing storage of the exact
+// size.
+func Concat[T any](slices ...[]T) []T {
+	length := 0
```

Comment:
nit: I think usually `var length int` is preferred in our codebase

Reply:
done